main.cpp

```diff
@@ -243,238 +242,229 @@ struct Resturant
         float countTip(float tip);
     };
 
-    //1) serve food
     void serveFood();
-    //2) collect payment
-    float collectPayement(float payment);
-    //3) provide eating utensils
+    float collectPayment(float payment);
     void utensils();
 };
-/*
-Thing 4) car dealership
-5 properties:
-    1) number of cars in inventory(int)
-    2) amount of cars sold in a month(int)
-    3) amount sales people on staff(int)
-    4) number of cars serviced each week(int)
-    5) amount of inventory bought this month(int)
-3 things it can do:
-    1) sell cars
-    2) hire salesmen
-    3) buy inventory
- */
+
+void Resturant::serveFood()
+{
+
+}
+
+float Resturant::collectPayment(float payment)
+{
+    return payment;
+}
+
+void Resturant::utensils()
+{
+
+}
+
+void Resturant::Server::serveCust()
+{
+
+}
```

Comment:
- Do not use abbreviations in function names.
`serveCustomer()` is what you need here.
- it's `Restaurant`, not `Resturant`
Correct the spelling of the class name.
- [ ] revise